meta-iot2000-example/recipes-node/node-red-dashboard/node-red-dashboard_2.23.4.bb

```diff
@@ -3,7 +3,7 @@ SUMMARY = "A set of dashboard nodes for Node-RED"
 # WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
 # your responsibility to verify that the values are complete and correct.
 LICENSE = "Apache-2.0 & MIT & ISC"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=f289d543031cf6f8f1943a54d087eeac"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=088ce976a5d78ef059391f1ca042eb3d"
```

Comment:
Same question here: New checksum == new licenses?

Reply:
Similar here, copyright year is changed from 2019 to 2020. Also, should we change the guessed LICENSE to more accurate "Apache-2.0"?

Reply:
Same as above.